### DISCUSSION TITLE
Dependent mixin implementation

### DISCUSSION THREAD

**Discussion by EranosyanEduard** [vuetifyjs/vuetify#20674] [Q&A] [answered]
Hello. 

You used the `Depend` mixin in _vuetify_ version 2 components (`VDialog`, `VMenu`, etc). 
I'd like to know to how did you stop using `Dependent` mixin in _vuetify_ version 3?

**Comment by KaelWD** [comment, marked as answer]
It was replaced with https://github.com/vuetifyjs/vuetify/blob/04c9699841afb86ceb9f0798f8d18a0123d3b0ac/packages/vuetify/src/composables/stack.ts

**Reply by EranosyanEduard** [reply]
Thank you.
